src/test/java/io/horrorshow/codey/time/DiscordTimezoneTest.java

```diff
@@ -0,0 +1,36 @@
+package io.horrorshow.codey.time;
```

Comment:
Add at least one more unit test that checks the `onMessage` method and if the embed is created with the right time. If you mock DiscordUtils to make this simpler a test like that could look like:
```java
    @Test
    void finds_time_in_message_and_replies_with_local_time() {
        var message = mock(Message.class, RETURNS_DEEP_STUBS);
        when(message.getContentRaw()).thenReturn("""
                this is a discord message, here
                is a time: Friday 15:00 Europe/Berlin cool cool
                here is some invalid time fRidai 24:60 Europe/Los_Angeles""");

        discordTimezone.onMessage(message);
        verify(utils).sendRemovableMessageReply(eq(message), embedCaptor.capture());
        var sentEmbed = embedCaptor.getValue();
        var time = sentEmbed.getTimestamp();
        assertThat(time).isNotNull();
        assertThat(time.getHour()).isEqualTo(15);
        assertThat(time.getMinute()).isZero();
        assertThat(time.getDayOfWeek()).isEqualTo(DayOfWeek.FRIDAY);
    }
```
you probably have to import a couple of classes for the `when`, `verify` and `assertThat`.
Since this test mocks the `DiscordUtils` you gotta set the test class up like that:
```java
    @Mock
    JDA jda;
    @Mock
    DiscordUtils utils;
    DiscordTimezone discordTimezone;
    @Captor
    ArgumentCaptor<MessageEmbed> embedCaptor;

    @BeforeEach
    void init() {
        MockitoAnnotations.openMocks(this);
        discordTimezone = new DiscordTimezone(jda, utils);
    }
```